src/components/AccountDefaultsSettings.vue

```diff
@@ -20,40 +20,28 @@
   -->
 
 <template>
-	<div class="section">
-		<h2>{{ t('mail', 'Defaults') }}</h2>
-		<p class="settings-hint">
-			{{
-				t('mail', 'Here you can select where Nextcloud Mail stores your drafts as well as sent and deleted messages.')
-			}}
-		</p>
-		<table>
-			<tr>
-				<td>
-					{{ t('mail', 'Draft messages are saved to:') }}
-				</td>
-				<td>
-					<MailboxInlinePicker v-model="draftsMailbox" :account="account" :disabled="saving" />
-				</td>
-			</tr>
-			<tr>
-				<td>
-					{{ t('mail', 'Sent messages are saved to:') }}
-				</td>
-				<td>
-					<MailboxInlinePicker v-model="sentMailbox" :account="account" :disabled="saving" />
-				</td>
-			</tr>
-			<tr>
-				<td>
-					{{ t('mail', 'Deleted messages are moved to:') }}
-				</td>
-				<td>
-					<MailboxInlinePicker v-model="trashMailbox" :account="account" :disabled="saving" />
-				</td>
-			</tr>
-		</table>
-	</div>
+	<table>
```

Comment:
Since we are now in a narrower modal with the dropdown below the text, instead of the whole table structure, a simple `<p>` for the text will do (followed by the MailboxInlinePicker). That way there’s also a bit of vertical spacing between the picker and the following text.